lib/platform/github/index.js

```diff
@@ -171,6 +172,7 @@ async function initRepo({
   config.repository = repository;
   [config.repositoryOwner, config.repositoryName] = repository.split('/');
   config.gitPrivateKey = gitPrivateKey;
+  config.optimizeForDisabled = optimizeForDisabled;
```

Comment:
If it's never used again then we don't need to save it to `config`

Reply:
agreed! Probably they won't be used again.